hyperspy/tests/signals/test_assign_subclass.py

```diff
@@ -141,27 +136,31 @@ def setup_method(self, method):
         self.s = hs.signals.Signal1D([0, 1])
 
     def test_lazy_to_eels_and_back(self):
+        exspy = pytest.importorskip("exspy")
```

Comment:
I think these tests are redundant, in hyperspy, we only need one test to check that setting a `signal_type` change the object class but that's all.

Testing assignment to signal specify class should go in `exSpy`